spec/feature_tests/user_stories_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'takeaway'
+require 'order'
+
+describe "User stories" do
+
+let(:takeaway) {Takeaway.new(menu, order)}
+let(:order) {Order.new(menu, text)}
+let(:menu) {Menu.new}
+let(:text) {Text.new}
+let(:menu_out) {"starter : 5\nmains : 10\ndesert : 3\n"}
+
+before do
+  allow(order).to receive(:text_confirmation)
+end
+
+  describe 'User Story 1' do
+
+    it 'allows a customer to see a list of dishes with prices' do
+      expect{takeaway.show_dishes}.to output(menu_out).to_stdout
+    end
+
+  end
+
+  describe 'User Story 2' do
+
+    it 'allows a customer to place an order' do
+      expect{takeaway.place_order("starter", 1, "desert", 3, 14)}.not_to raise_error
```

Comment:
Metrics/LineLength: Line is too long. [84/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)